### DISCUSSION TITLE
POINT 6166

### DISCUSSION THREAD

**Discussion by AngieFloresG** [cloudsen12/dataset#21] [Lissette]
20190317T142751_20190317T143036_T20MNC
![image](https://user-images.githubusercontent.com/78394685/146701905-c5ec8642-3cd6-4499-8c9d-9f13515da861.png)

20190715T142801_20190715T142758_T20MNC
![image](https://user-images.githubusercontent.com/78394685/146702095-5b8f8612-6523-4a59-8af0-98df35c0f357.png)
 
20200111T142701_20200111T142659_T20MNC 
pixeles sueltos
![image](https://user-images.githubusercontent.com/78394685/146702168-a5d1eb7b-3730-4fa0-9d74-d9a8a362d6e4.png)

20200311T142731_20200311T142727_T20MNC
![image](https://user-images.githubusercontent.com/78394685/146702293-246e43b1-d6a9-441d-9911-bf79562999bd.png)

@LissADiaz

**Comment by csaybar** [comment]
**20190317T142751_20190317T143036_T20MNC**

![image](https://user-images.githubusercontent.com/16768318/146845411-dbd8f8e6-3fbe-49a8-a037-b7ea9d66fc41.png)
